community/event_management/doctype/exhibitor_registration/exhibitor_registration.json

```diff
@@ -0,0 +1,55 @@
+{
+ "actions": [],
+ "autoname": "Exhibitor-Regis-.####.",
+ "creation": "2021-08-16 16:26:46.189119",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "full_name",
+  "company",
+  "logo"
+ ],
+ "fields": [
+  {
+   "fieldname": "full_name",
+   "fieldtype": "Data",
+   "label": "Full Name"
+  },
+  {
+   "fieldname": "company",
+   "fieldtype": "Data",
+   "label": "Company ",
+   "unique": 1
```

Comment:
We shouldn't be keeping this unique. What if 2-3 people from the same company whats to register to exhibit different stuff.